[docs] #85: Add UML diagrams, create a new chapter about objects (#148)
hyperledger/iroha-2-docs: src/guide/objects/hierarchy.md

| Kotlin/Java           | Register a [domain](../kotlin-java.md#3-registering-a-domain), an [account](../kotlin-java.md#4-registering-an-account), an [asset](../kotlin-java.md#5-registering-and-minting-assets) |
| Python                | Register a [domain](../python.md#3-registering-a-domain), an [account](../python.md#4-registering-an-account), an [asset](../python.md#5-registering-and-minting-assets)                |
| JavaScript/TypeScript | Register a [domain](../javascript.md#3-registering-a-domain), an [account](../javascript.md#4-registering-an-account), an [asset](../javascript.md#5-registering-and-minting-assets)    |

:::

The diagram below provides a more detailed illustration of the relationship
between domains, accounts, and assets in the blockchain. You can learn more
about [permissions and roles](../advanced/permissions.md) and
[metadata](metadata.md) in the corresponding sections of the tutorial. The
asset structure is illustrated in a [dedicated chapter](./assets.md).

::: info

Note that the diagram below depicts Account structure for the LTS version

> of Iroha. In the stable version, permission tokens are stored not within

Reviewer: ```suggestion
of Iroha. In the stable version, permission tokens aren't stored within
```
